ui/pages/Search/constants.js

```diff
@@ -187,30 +187,62 @@ export const STAFF_PATHOGENICITY_FIELDS = [
   },
 ]
 
-export const STAFF_PATHOGENICITY_FILTER_OPTIONS = [
-  {
+export const ALL_PATHOGENICITY_FILTER = 'any'
+export const PATH_LIKELY_PATH_FILTER = 'pathogenic_likely_pathogenic'
+export const NOT_BENIGN = 'not_benign'
+
+export const STAFF_PATHOGENICITY_FILTER_LOOKUP = {
+  [ALL_PATHOGENICITY_FILTER]: {
+    text: 'Any',
+    filter: {
+      [CLINVAR_NAME]: [],
+      [HGMD_NAME]: [],
+    },
+  },
+  [PATH_LIKELY_PATH_FILTER]: {
     text: 'Pathogenic/ Likely Path.',
-    value: {
+    filter: {
       [CLINVAR_NAME]: [CLIVAR_PATH, CLINVAR_LIKELY_PATH],
       [HGMD_NAME]: [HGMD_DM],
     },
   },
-  {
+  [NOT_BENIGN]: {
     text: 'Not Benign',
-    value: {
+    filter: {
       [CLINVAR_NAME]: [CLIVAR_PATH, CLINVAR_LIKELY_PATH, CLINVAR_UNCERTAIN],
       [HGMD_NAME]: HGMD_OPTIONS.map(({ value }) => value),
     },
   },
-]
+}
+
+export const STAFF_PATHOGENICITY_FILTER_OPTIONS = [
+  ALL_PATHOGENICITY_FILTER, PATH_LIKELY_PATH_FILTER, NOT_BENIGN,
+].map(value => ({ value, ...STAFF_PATHOGENICITY_FILTER_LOOKUP[value] }))
+
```

Comment:
In most places we define the `OPTIONS` as a list, and then we do a reduce to get the `LOOKUP`, rather than what you have done here which is the reverse, so this is kind of an anti-pattern. We only do the defined exported constants for specific filters if you actually need to be able to import them by themselves in other components, which you shouldn't need to do for most of these. What you should do here is leave the `STAFF_PATHOGENICITY_FILTER_OPTIONS` the same, and then do something like
```
export const STAFF_PATHOGENICITY_FILTER_LOOKUP = STAFF_PATHOGENICITY_FILTER_OPTIONS.reduce(
  (acc, option) => ({ ...acc, [option.value]: option }), {}
)
```